cmd/cloud/deployment.go

```diff
@@ -289,7 +292,7 @@ func deploymentCreate(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-	return deployment.Create(label, workspaceID, description, clusterID, runtimeVersion, schedulerAU, schedulerReplicas, astroClient, waitForStatus)
+	return deployment.Create(label, workspaceID, description, clusterID, runtimeVersion, dagDeploy, schedulerAU, schedulerReplicas, astroClient, waitForStatus)
```

Comment:
nitpick: if we are going with strings instead of bool, then having the conversion upfront over here itself to boolean could simplify the whole logic inside `cloud` pkg